sdk/src/client/bindings/python/iota_client/client.py

```diff
@@ -382,4 +526,334 @@ def as_dict(self):
 
 class CoinType(Enum):
     IOTA = 4218
-    SHIMMER = 4219
\ No newline at end of file
+    SHIMMER = 4219
+
+class UnlockConditionType(Enum):
+    Address = 0
+    StorageDepositReturn = 1
+    Timelock = 2
+    Expiration = 3
+    StateControllerAddress = 4
+    GovernorAddress = 5
+    ImmutableAliasAddress = 6
+
+class UnlockCondition():
+    def __init__(self, type=None, address=None, amount=None, unix_time=None, return_address=None) -> None:
```

Comment:
I know. That's why we have the wrapper classes now, right? :trollface: 

The "problem" is: I didn't want to let the bindings decide what is possible and what not, as much as possible. Because I was afraid that this could lead to an problem where we get the native rust implementation and the bindings out of sync 🤔